src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.test.jsx

```diff
@@ -1,6 +1,6 @@
-import 'CourseAuthoring/editors/setupEditorTest';
```

Comment:
Oh - this is the problem that's causing all the changes in the snapshot test. Is there some way you can leave `setupEditorTest` in place ?

Reply:
Nice catch! Added in  https://github.com/openedx/frontend-app-authoring/pull/1825/commits/58c14671d9a0b539ebadc1837005626997283281